indico/modules/events/abstracts/templates/management/abstract_list_filter.html

```diff
@@ -0,0 +1,90 @@
+{% from 'forms/_form.html' import form_header, form_footer, form_rows %}
+
+{% macro _render_column_selector(item_id, item, filter_choices, is_static_item=false) %}
+    {% set filters = filters['items'] if is_static_item else filters['fields'] %}
+    <div class="label-group list-column">
+        <div class="i-label title-wrapper" data-id="{{ item_id }}">
+            <div class="title">{{ item.title }}</div>
+            <div class="actions">
+                <span class="visibility icon-eye"></span>
+            </div>
+        </div>
+        {% if filter_choices %}
+            <div class="i-label filter">
+                <a class="icon-filter" data-toggle="dropdown"></a>
+                <ul class="dropdown" style="display: none">
+                    {% for value, caption in filter_choices.iteritems() %}
+                        <li>
+                            <input type="checkbox" name="field_{{ item_id }}" value="{{ value }}"
```

Comment:
Maybe not right now, but this template is almost identical to `reglist_filter.html`. Probably we can add a base template for both of them and then override it to specify the few things that are different.